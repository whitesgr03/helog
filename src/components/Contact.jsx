import style from "../styles/Contact.module.css";

import Address from "../components/Address";

const Contact = () => {
	return (
		<div className={style.contact}>
			<h3>Contact</h3>
			<p>Please contact us, If you have any questions.</p>
			<div className={style.container}>
				<Address />
			</div>
		</div>
	);
};

export default Contact;
