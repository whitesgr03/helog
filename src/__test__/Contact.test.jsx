import { vi, describe, it, expect } from "vitest";
import { render } from "@testing-library/react";

import Contact from "../components/Contact";

vi.mock("../components/Address.jsx", () => ({
	default: () => <div>Address component</div>,
}));

describe("Contact component", () => {
	it("should render correctly", () => {
		const { asFragment } = render(<Contact />);

		const actual = asFragment();

		expect(actual).toMatchSnapshot();
	});
});
