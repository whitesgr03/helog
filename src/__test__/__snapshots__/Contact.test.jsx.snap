// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > should render correctly 1`] = `
<DocumentFragment>
  <div
    class="_contact_2b0bd8"
  >
    <h3>
      Contact
    </h3>
    <p>
      Please contact us, If you have any questions.
    </p>
    <div
      class="_container_2b0bd8"
    >
      <div>
        Address component
      </div>
    </div>
  </div>
</DocumentFragment>
`;
